roles/ceph_provisioners/vars/main.yml

```diff
@@ -30,3 +30,6 @@ _ceph_provisioners_helm_values:
     job_namespace_client_key_cleaner: false
     job_namespace_client_key: false
     storageclass: false
+  pod:
+    labels:
+      include_app_kubernetes_io: false
```

Comment:
To keep things consistent, the pod section should be above the conf section like the others.